doc/api/yonomi-sdk/DevicesRepository/responseToDeviceTraitConverter.md

```diff
@@ -39,12 +39,13 @@ static List<Trait> responseToDeviceTraitConverter(dynamic deviceTraits) {
           .contains('thermostat_setting')) {
         return ThermostatTrait(
             'thermostat_setting',
-            TargetTemperature((trait
-                    as GgetDeviceData_device_traits__asThermostatSettingDeviceTrait)
-                .state
-                .targetTemperature
-                .reported
-                ?.value));
+            TargetTemperature(
+                (trait as GgetDeviceData_device_traits__asThermostatSettingDeviceTrait)
+                        .state
+                        .targetTemperature
+                        .reported
+                        ?.value ??
+                    0.0));
```

Comment:
Not sure if it would matter or not but should 0 be the default target temp?

Reply:
I guess it would be one of those biz call decisions. Plus would it be F or C or K? Who knows. 0.0 could make sense for some sort of smart freezer but not for a house thermostat.

Reply:
I had to put this to make Virtual Connector work with a few of these null values